### PR TITLE
test(v2-ontology-cache): add test for isEditable property on link values (DSP-1840)

### PR DESCRIPTION
**Pull request by subotic**
resolves DSP-1840

### CODE REVIEW

File: webapi/src/test/scala/org/knora/webapi/responders/v2/ontology/CacheSpec.scala [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+      val ontologiesFromCacheFuture: Future[Map[SmartIri, ReadOntologyV2]] = for {
+        cacheData: Cache.OntologyCacheData       <- Cache.getCacheData
+        ontologies: Map[SmartIri, ReadOntologyV2] = cacheData.ontologies
+      } yield ontologies
+
+      ontologiesFromCacheFuture map { ontologies: Map[SmartIri, ReadOntologyV2] =>
+        ontologies.size should equal(13)
+      // TODO: check loaded data for correctness
+      }
+      assert(true)
```
Comment (subotic) [inline webapi/src/test/scala/org/knora/webapi/responders/v2/ontology/CacheSpec.scala:104]: this is not testing anything. do we need this here?

File: webapi/src/test/scala/org/knora/webapi/e2e/v2/tempE2eSpec2.scala [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+import org.knora.webapi.messages.v2.responder.ontologymessages.{InputOntologyV2, TestResponseParsingModeV2}
+import org.knora.webapi.messages.{OntologyConstants, StringFormatter}
+import org.knora.webapi.routing.v2.OntologiesRouteV2
+import org.knora.webapi.sharedtestdata.{SharedOntologyTestDataADM, SharedTestDataADM}
+
+import java.net.URLEncoder
+import java.time.Instant
+import scala.concurrent.ExecutionContextExecutor
+
+object tempE2eSpec2 {
```
Comment (subotic) [inline webapi/src/test/scala/org/knora/webapi/e2e/v2/tempE2eSpec2.scala:21]: do we need tempE2ESpec2?

File: webapi/src/test/scala/org/knora/webapi/e2e/v2/tempE2eSpec.scala [hunk 3]
```diff
@@ -0,0 +1,451 @@
...
+object tempE2eSpec {
+  private val anythingUserProfile = SharedTestDataADM.anythingAdminUser
+  private val anythingUsername    = anythingUserProfile.email
+  private val password            = SharedTestDataADM.testPass
+}
+
+/**
+ * End-to-end test specification for API v2 ontology routes.
+ */
+class tempE2eSpec extends R2RSpec {
```
Comment (subotic) [inline webapi/src/test/scala/org/knora/webapi/e2e/v2/tempE2eSpec.scala:33]: do we need tempE2eSpec?

File: webapi/src/test/scala/org/knora/webapi/responders/v2/ontology/CacheSpec.scala [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+      ontologiesFromCacheFuture map { res: Map[SmartIri, ReadOntologyV2] => res.size should equal(13) }
+    }
+
+    "successfully load back all ontologies from cache" in {
+      val ontologiesFromCacheFuture: Future[Map[SmartIri, ReadOntologyV2]] = for {
+        cacheData: Cache.OntologyCacheData       <- Cache.getCacheData
+        ontologies: Map[SmartIri, ReadOntologyV2] = cacheData.ontologies
+      } yield ontologies
+
+      ontologiesFromCacheFuture map { ontologies: Map[SmartIri, ReadOntologyV2] =>
```
Comment (subotic) [inline webapi/src/test/scala/org/knora/webapi/responders/v2/ontology/CacheSpec.scala:100]: is this running correctly or do we need asyncSpec?

File: webapi/src/test/scala/org/knora/webapi/responders/v2/ontology/CacheSpec.scala (hunk 1 above)
Comment (BalduinLandolt) [reply webapi/src/test/scala/org/knora/webapi/responders/v2/ontology/CacheSpec.scala]: removed

File: webapi/src/test/scala/org/knora/webapi/e2e/v2/tempE2eSpec2.scala (hunk 2 above)
Comment (BalduinLandolt) [reply webapi/src/test/scala/org/knora/webapi/e2e/v2/tempE2eSpec2.scala]: no... removed

File: webapi/src/test/scala/org/knora/webapi/e2e/v2/tempE2eSpec.scala (hunk 3 above)
Comment (BalduinLandolt) [reply webapi/src/test/scala/org/knora/webapi/e2e/v2/tempE2eSpec.scala]: removed

File: webapi/src/test/scala/org/knora/webapi/responders/v2/ontology/CacheSpec.scala (hunk 4 above)
Comment (BalduinLandolt) [reply webapi/src/test/scala/org/knora/webapi/responders/v2/ontology/CacheSpec.scala]: we checked, it seems to work

**Review by subotic — COMMENTED** [review]
LGTM, thanks!

**Review by irinaschubert — APPROVED** [review]

**Review by BalduinLandolt — APPROVED** [review]

**Pull request merged**